### PR TITLE
Add ssl option to RemoteDatabaseConfig

### PR DESCRIPTION
**Pull request by gumigumi4f**
We are trying to connect background postgresql database with server and client certificates. (Ex: Google Cloud SQL)
But there is no option to change sslfactory and sslmode and we cannot connect database.

After merging this PR, we are able to connect postgresql with certificates by adding the following settings and placing the certificates to `${user.home}/.postgresql/`.
```
database.ssl = true
database.sslfactory = org.postgresql.ssl.LibPQFactory
database.sslmode = verify-ca
```

### CODE REVIEW

**Comment by yoyama** [comment]
@gumigumi4f Thank you for your contribution.
I think we should update the document for new configuration.
https://docs.digdag.io/command_reference.html#server
You can update the doc in `digdag-docs` directory.

**Review by yoyama — CHANGES_REQUESTED** [review]
As comment, please update document.

**Comment by gumigumi4f** [comment]
Sorry for late response.
I have added new option such as sslcert and updated digdag-docs.

File: digdag-docs/src/command_reference.rst [hunk 1]
```diff
@@ -361,6 +361,11 @@ In the config file, following parameters are available
 * database.loginTimeout (seconds in integer, default: 30)
 * database.socketTimeout (seconds in integer, default: 1800)
 * database.ssl (boolean, default: false)
+* database.sslfactory (string, default: org.postgresql.ssl.NonValidatingFactory)
+* database.sslmode (string)
```
Comment (yoyama) [inline digdag-docs/src/command_reference.rst:365]: Could you describe what string are accepted for this option ?

File: digdag-docs/src/command_reference.rst [hunk 2]
```diff
@@ -361,6 +361,11 @@ In the config file, following parameters are available
...
+* database.sslcert (string)
```
Comment (yoyama) [inline digdag-docs/src/command_reference.rst:366]: Is this a path to cert file ?

File: digdag-docs/src/command_reference.rst [hunk 3]
```diff
@@ -361,6 +361,11 @@ In the config file, following parameters are available
...
+* database.sslkey (string)
+* database.sslrootcert (string)
```
Comment (yoyama) [inline digdag-docs/src/command_reference.rst:368]: ditto

**Comment by gumigumi4f** [comment]
I have updated the docs

**Comment by yoyama** [comment]
Thank you for your contribution. After CI has passed, I will approve it.

**Review by yoyama — APPROVED** [review]
LGTM 👍 
Thank you for your contribution

**Pull request merged**